std/fs/walk.ts

```diff
@@ -4,6 +4,7 @@
 import { assert } from "../_util/assert.ts";
 import { basename, join, normalize } from "../path/mod.ts";
 
+/** Create WalkEntry for the `path` synchronously */
```

Comment:
Excuse me, where did you find these comments?
https://deno.land/std@0.78.0/fs/walk.ts ?
I didn't find it.